README.md

```diff
@@ -27,7 +27,9 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
 ```ruby
-# code here
+def offer_rose(person)
+  puts "Would you take this rose," person ", in exchange for giving an old beggar woman shelter from the bitter cold?"
+end
 ```
```

Comment:
think about how you would get the value in person to display. review string interpolation `#{person}`